Bottom: 8056c2216d8f45527f3b52e0c40c1491c0aad2e0
Top:    b4938566bb14b3b579a08eee0461b382d9e6fbcc
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-18 17:14:41 +0800

Refresh of vhost_net-rename

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index ed21ce1..d8b9e37 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -700,13 +700,10 @@ static void handle_tx(struct vhost_net *net)
 	vhost_disable_notify(&net->dev, vq);
 	vhost_net_disable_vq(net, vq);
 
-	if (vhost_sock_zcopy(sock)) {
-		printk("zcopy!\n");
+	if (vhost_sock_zcopy(sock))
 		handle_tx_zerocopy(net, sock);
-	} else {
-		printk("data copy!\n");
+	else
 		handle_tx_copy(net, sock);
-	}
 
 out:
 	mutex_unlock(&vq->mutex);
